app/Http/Controllers/ApiPhotosController.php

```diff
@@ -256,7 +256,12 @@ public function addTags (AddTagsApiRequest $request)
             'request' => $request->all()
         ]);
 
-        dispatch (new AddTags($user->id, $photo->id, $request->litter ?? $request->tags));
+        dispatch (new AddTags(
+            $user->id,
+            $photo->id,
+            $request->litter ?? $request->tags,
+            $request->picked_up
```

Comment:
what happens if an older app version hits this request and they don't have picked up in the request?

Reply:
If you don't provide this parameter, the picked-up value will default to the user's predefined setting for presence. This is the case for current and older app versions that don't provide this value.